tests/Memed/Soluti/SignerTest.php

```diff
@@ -9,6 +9,7 @@
 use Memed\Soluti\Auth\CloudAuthentication;
 use Memed\Soluti\Auth\Credentials;
 use Memed\Soluti\Auth\Session;
+use Memed\Soluti\Auth\Token;
 use Memed\Soluti\Auth\Token as AuthToken;
```

Comment:
Remove this unnecessary/duplicated class. Choose between `Token` or `AuthToken`

Reply:
Done.